### PR TITLE
Task one with all components

### PR DESCRIPTION
**Pull request by sameerdutta9703**

### CODE REVIEW

File: src/app/app.component.html [hunk 1]
```diff
@@ -1,534 +1,11 @@
...
-<!-- * * * * * * * * * * and can be replaced. * * * * * * * * * * * -->
-<!-- * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * -->
-<!-- * * * * * * * * * * End of Placeholder * * * * * * * * * * * -->
-<!-- * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * -->
-
-
-
-<router-outlet></router-outlet>
\ No newline at end of file
+<div class="componenet-div">
```
Comment (itaphuviinnay) [inline src/app/app.component.html]: Not a good practice to name css classes with `div`. You can use `app-components-container`

File: src/app/app.component.html [hunk 2]
```diff
@@ -1,534 +1,11 @@
...
-
-      <span>Angular Blog</span>
-
-      <svg class="material-icons" xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"><path d="M10 6L8.59 7.41 13.17 12l-4.58 4.59L10 18l6-6z"/></svg>
-    </a>
-
+<div class="container">
+  <div class="jumbotron">
+    <h1 class="display-3">Hello, Vinay!</h1>
+    <p class="lead" appHighlight>Created this component manually for fun</p>
```
Comment (itaphuviinnay) [inline src/app/app.component.html]: What is `appHighlight` ?

Comment (sameerdutta9703) [reply src/app/app.component.html]: Screenshot (6)

Was trying something like this

File: src/app/app.component.html (hunk 1 above)
Comment (sameerdutta9703) [reply src/app/app.component.html]: changed the name of the div

File: src/app/app.component.html (hunk 2 above)
Comment (itaphuviinnay) [reply src/app/app.component.html]: Really nice that you are going through the official docs. But at the same time this task doesn't include directives. So you can remove that and use it for the upcoming directives task.

Comment (sameerdutta9703) [reply src/app/app.component.html]: ok sure, removed those directives

**Pull request merged**